src/main/java/core/basesyntax/dao/processing/SortingService.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.dao.processing;
+
+import core.basesyntax.products.Fruit;
+import core.basesyntax.storage.ListStorage;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class SortingService {
+    public List<String> sortDataBeforeWrite() {
```

Comment:
Move this method to storage. Not sure we need a separate class for it

Reply:
Moved to Storage, fixed tests as well.